enterprise/backend/src/metabase_enterprise/data_editing/scope.clj

```diff
@@ -0,0 +1,77 @@
+(ns metabase-enterprise.data-editing.scope
+  (:require
+   [metabase-enterprise.data-editing.types :as types]
+   [metabase.util.malli :as mu]
+   [toucan2.core :as t2]))
+
+;; TODO Perhaps we will prefer the FE just sending this explicitly instead?
+;; For now, it is not doing this, so we infer it.
+(mu/defn scope-type
+  "Classify the scope, useful for switching on in logic and templates."
+  [scope :- ::types/scope.raw] :- :keyword
+  (condp contains? scope
+    :dashcard-id :dashcard
+    :dashboard-id :dashboard
+    :card-id :card
+    :table-id :table
+    :webhook-id :webhook))
```

Comment:
cannot believe this man told me something about vertical align once 😁

Reply:
Haha I blame dumb Cursive autoformat